changelog.txt

```diff
@@ -1,5 +1,18 @@
 == Changelog ==
 
+= 0.5.0 =
+* Fix redirect CPT to be private
+* Add filters to modify the contents of the HTTP Status Code drop down enhancement good first issue
+* Sanitise full from URL to relative path
+* Fix redirects with query arguments bug
```

Comment:
What was the bug?

Reply:
I'll add issue numbers - essentially it was doubling up on the query string eg. `/?this=that?this=that`